src/components/footer.css

```diff
@@ -19,10 +19,22 @@
   margin-top: 16px;
 }
 
+.footer-icons img {
+  transition: box-shadow 0.1s ease-out, transform 0.1s ease-out;
+  box-shadow: 4px 4px #f05657;
+  border-radius: 50%;
+  border: 1px solid #f05657;
+}
+
 .footer-icons > a {
   margin-right: 16px;
 }
 
+.footer-icons a:hover img {
+  transform: translate3d(3px, 3px, 0);
```

Comment:
Could you actually remove the hover transform? I'll be updating the hover state for all buttons now! `cursor: pointer` will do on the base element!

Thanks!!